javaparser-core/src/main/java/com/github/javaparser/JavaParser.java

```diff
@@ -336,6 +340,7 @@ public static Statement parseStatement(final String statement) {
     private static <T extends Node> T simplifiedParse(ParseStart<T> context, Provider provider) {
         ParseResult<T> result = new JavaParser(staticConfiguration).parse(context, provider);
         if (result.isSuccessful()) {
+            considerInjectingSymbolResolver(result, staticConfiguration);
```

Comment:
This is already done in the other method.

Reply:
Right, my bad